run/make_init25kmNH.py

```diff
@@ -160,6 +160,7 @@ def topaz4_interpolate(target_lon_deg, target_lat_deg, data, lat_array):
     mask = datagrp.createVariable("mask", "f8", field_dims)
     sst_data = topaz4_interpolate(element_lon, element_lat, source_file["temperature"][0, :, :].squeeze(), lat_array)
     mask[:, :] = 1 - ma.getmask(sst_data)
+    mask[100, 95] = 0.0 # Ad-hoc fix for the westernmost point of the New Siberian Islands
```

Comment:
is this safe to leave in?

Reply:
Yes, but perhaps it should not be in this PR.

For a reason I cannot fathom, the (apparently) same code working on the same data consistently produces a difference in the land-sea mask between the forcing files and this initial restart file. The difference is whether this single point is land or sea. Adding this change makes the two datasets consistent.

Of course, it would be better if the two python files produced the same data, but I have not been able to get to that point.

Since it does not actually concern the single-column thermodynamics tests, this change shouldn't really be in this PR.

Reply:
I think I misunderstood your comment:

> Since it does not actually concern the single-column thermodynamics tests, this change shouldn't really be in this PR.

Do this mean leave it in this PR or take it out of this PR?

Reply:
I'll create another PR such that is becomes redundant in this PR.

Reply:
See #441 

We should merge #441 before this PR. Do we need to rebase this PR for one line in a python script?

Reply:
PR #441 has been merged.